website/integrations/services/apple/index.md

```diff
@@ -0,0 +1,385 @@
+
+### 1. Create scope mappings
+
+#### 1.1. Mapping the user's profile information
+
+From the authentik Admin interface, navigate to **Customization** -> **Property Mappings** and click **Create**.
+When prompted for a choice of mapping, select **Scope Mapping** and continue with the following form values:
+
+| Field      | Value                            |
+| ---------- | -------------------------------- |
+| Name       | `Apple Business Manager profile` |
+| Scope Name | `profile`                        |
+
+This scope mapping will also require an expression to map Apple Business Manager's name field into into separate given and family names. This expression assumes that the name is formatted with the given name first, followed by the family name, delimited by a space. Consider adjusting the expression to match the name format used in your organization.
+
+```py title="Expression field: Extracting given and family names."
+given_name, _, family_name = request.user.name.partition(" ")
+
+return {
+  "given_name": given_name,
+  "family_name": family_name,
+}
+```
+
+Click **Finish** and confirm that new scope mapping is listed in the **Property Mappings** overview.
+
+#### 1.2. Provide read access for the scope mapping
+
+From the **Property Mappings** overview, click **Create**.
+
+When prompted for a choice of mapping, select **Scope Mapping** and continue with the following form values:
+
+| Field      | Value                             |
+| ---------- | --------------------------------- |
+| Name       | `Apple Business Manager ssf.read` |
+| Scope Name | `ssf.read`                        |
+| Expression | `return {}`                       |
+
+Click **Finish** and confirm that new scope mapping is listed in the **Property Mappings** overview.
+
+#### 1.3. Provide management access for the scope mapping
+
+From the **Property Mappings** overview, click **Create**.
+When prompted for a choice of mapping, select **Scope Mapping** and continue with the following form values:
+
+| Field      | Value                               |
+| ---------- | ----------------------------------- |
+| Name       | `Apple Business Manager ssf.manage` |
+| Scope Name | `ssf.manage`                        |
+| Expression | `return {}`                         |
+
+Click **Finish** and confirm that all three new scope mappings are listed in the overview.
+
+### 2. Create signing keys
+
+You will need to create **Signing Key** to sign Security Event Tokens (SET). This key is used to both sign and verify the SETs that are sent between authentik and Apple Business Manager.
+
+From the Admin interface, navigate to **System** -> **Certificates** and clicking **Generate** or **Create**.
+When choosing a **Common Name**, consider using a name that reflects the purpose of the key, such as `apple-business-manager`.
+
+### 3. Create OIDC provider
```

Comment:
So everything in this section should be a series of steps, that reflects as closely as possible the exact steps that the user will take in the UI, as they create a provider. We don't need substeps... just the steps on each panel of the create modal.